apis/frontend/react-support/README.md

```diff
@@ -1,3 +1,262 @@
 # Nosto React
 
-Dedicated support for React is provided via the [nosto-react](https://github.com/Nosto/nosto-react) library. nosto-react is a thin layer of React components and hooks on top of the Session API.
\ No newline at end of file
+[Nosto React](https://github.com/Nosto/nosto-react) is a React component library to make it even easier to implement Nosto.
+
+The library provides you everything to get started with personalization on your React site. It's dead simple, and beginner friendly.
+
+### Why?
+
+You should be using Nosto React if you want to:
+
+- Design personalization experiences with best practices
+- Customize your components at will and with ease
+- Follow React principles
+
+## Feature list
+
+Our React component library includes the following features:
+
+- Recommendations, including client-side rendering
+- Onsite content personalization
+- Dynamic bundles
+- Debug toolbar\* (excluding advanced use cases)
+- Pop-ups & personalised emails
+- A/B testing
+- Segmentation and Insights
+- Analytics
+- Search\*\* (when implemented via our code editor)
+
+_\*Note: Our React component library currently does not support advanced use cases of the debug toolbar, but we are constantly working to improve our library and provide you with the best possible integration options. We support most use-cases with page tagging and debug workflows._
+
+_\*\*Note: The search feature is available when implemented via our code editor._
+
+### Additional features
+
+- ✓ Lightweight. Almost zero bloat.
+- ✓ Full support for the Facebook Pixel and Google Analytics.
+- ✓ Full support for leveraging overlays.
+- ✓ Full support for the JS API.
+- ✓ Full support for placements.
```

Comment:
Nitpick:
Do we need both bullets and checkmark here? Shall we considering using one?